modules/cms/src/helpers/BlockHelper.php

```diff
@@ -139,6 +139,19 @@ public static function imageArrayUpload($value, $applyFilter = false, $returnObj
      * ```php
      * 'file' => BlockHelper::FileUpload($this->getVarValue('myFile')),
      * ```
+     * Attention: Always use if statement in your view file to check if file exists.
+     *
+     * Example in block extrasVars():
```

Comment:
its:  extraVars() not extrasVars().
Maybe instead of repeat the above comment just extend the previous comment block by:

 extraVars() 
{
    'file' => => ...
}

so then you can refere in your "attention" block to the above already defined file extraVar.